docs/source/features/parallelisms.rst

```diff
@@ -1,56 +1,48 @@
 .. _parallelisms:
 
 Parallelisms
-------------
+============
 
-NeMo Megatron supports five types of parallelism (which can be mixed together arbitrarily).
+NeMo Megatron supports various data- and model-parallel deep learning workload deployment methods (which can be mixed together arbitrarily).
 
 Data Parallelism
-^^^^^^^^^^^^^^^^
+----------------
 
-Data Parallelism (DP) creates identical copies of the model across
-multiple GPUs. Data batches are distributed between GPUs so that the
-GPUs can process them independently. While compute is efficiently
-distributed between GPUs, communication is required in order to keep
-the model copies consistent with each other.
+Data Parallelism (DP) replicates the model across multiple GPUs.
+Data batches are evenly distributed between GPUs and the data-parallel GPUs process them independently.
+While the computation workload is efficiently distributed across GPUs, inter-GPU communication is required in order to keep the model replicas consistent between training steps.
 
 Distributed Data Parallelism
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Distributed Data Parallelism (DDP) keeps model copies consistent by
-synchronizing parameter gradients before each optimization step. More
-specifically, it sums gradients over all model copies using an
-all-reduce communication collective.
+Distributed Data Parallelism (DDP) keeps the model copies consistent by synchronizing parameter gradients across data-parallel GPUs before each parameter update.
+More specifically, it sums the gradients of all model copies using all-reduce communication collectives.
 
 .. image:: ../nlp/nemo_megatron/images/ddp.gif
     :align: center
     :width: 800px
     :alt: Distributed Data Parallel
 
-Distributed Optimizer (ZeRO-1)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Distributed Optimizer
```

Comment:
is there a reason of removing zero1?

Reply:
I removed it because `ZeRO-1` is the term used in Deepspeed, which we don't explain what `ZeRO` means. I think all the descriptions we have here should explain what the Distributed Optimizer is.